### PR TITLE
dep: upgrade github.com/dgryski/go-farm

### PR DESCRIPTION
**Pull request by lance6716**
### What problem does this PR solve?

Issue Number: ref #56635

Problem Summary:

### What changed and how does it work?

just check if the problem will gone, because I don't know the root cause 😢 

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -221,6 +221,8 @@ github.com/dgryski/go-farm v0.0.0-20190104051053-3adb47b1fb0f/go.mod h1:SqUrOPUn
 github.com/dgryski/go-farm v0.0.0-20190423205320-6a90982ecee2/go.mod h1:SqUrOPUnsFjfmXRMNPybcSiG0BgUW2AuFH8PAnS2iTw=
 github.com/dgryski/go-farm v0.0.0-20200201041132-a6ae2369ad13 h1:fAjc9m62+UWV/WAFKLNi6ZS0675eEUC9y3AlwSbQu1Y=
 github.com/dgryski/go-farm v0.0.0-20200201041132-a6ae2369ad13/go.mod h1:SqUrOPUnsFjfmXRMNPybcSiG0BgUW2AuFH8PAnS2iTw=
```
Comment (Defined2014) [inline go.sum:223]: Also included by other pkgs :(

**Review by Defined2014 — APPROVED** [review]

File: go.sum (hunk 1 above)
Comment (lance6716) [reply go.sum:223]: I'll update them gradually 😢

**Review by hawkingrei — APPROVED** [review]

File: go.sum (hunk 1 above)
Comment (lance6716) [reply go.sum:223]: I need to update https://github.com/pingcap/badger and https://github.com/tikv/client-go. Other go.sum items come from unit test files

**Pull request merged**